From 99775df8356ab6c7dd77493b35e59de5f6a773f8 Mon Sep 17 00:00:00 2001
From: Christy Norman <christy@linux.vnet.ibm.com>
Date: Wed, 4 Sep 2019 18:41:37 -0400
Subject: [PATCH] fix build for Power

The zip -d flag will error if it doesn't find that extension.
This is a fix for Power, but doesn't fix the default case.

Signed-off-by: Christy Norman <christy@linux.vnet.ibm.com>
---
 src/conditions/BUILD | 6 ++++++
 third_party/BUILD    | 1 +
 2 files changed, 7 insertions(+)

diff --git a/src/conditions/BUILD b/src/conditions/BUILD
index 2b28e280576..faa41a439d4 100644
--- a/src/conditions/BUILD
+++ b/src/conditions/BUILD
@@ -10,6 +10,12 @@ filegroup(
     visibility = ["//src:__pkg__"],
 )
 
+config_setting(
+    name = "linux_ppc",
+    values = {"cpu": "ppc"},
+    visibility = ["//visibility:public"],
+)
+
 config_setting(
     name = "linux_x86_64",
     values = {"cpu": "k8"},
diff --git a/third_party/BUILD b/third_party/BUILD
index 7545b3df33a..b4115b2988c 100644
--- a/third_party/BUILD
+++ b/third_party/BUILD
@@ -528,6 +528,7 @@ UNNECESSARY_DYNAMIC_LIBRARIES = select({
     # The .so file is an x86 one, so we can just remove it if the CPU is not x86
     "//src/conditions:arm": "*.so *.jnilib *.dll",
     "//src/conditions:linux_aarch64": "*.so *.jnilib *.dll",
+    "//src/conditions:linux_ppc": "*.so *.jnilib *.dll",
     # Play it safe -- better have a big binary than a slow binary
     # zip -d does require an argument. Supply something bogus.
     "//conditions:default": "*.bogusextension",
